### ISSUE TITLE
Sidebar menus have too much spacing

### ISSUE THREAD

**Issue by vj777** [mpcjanssen/simpletask-android#703]
You can only have about 8 contexts or project categories before you have to scroll on a phone.  Each line for context or project quick filter takes up about twice as much vertical space as it needs to.  If the line spacing was reduced a lot more categories could be shown. 
I appreciate it might be a concern for those with fat fingers, but I think there is way too much room at present.  It's even worse in landscape view.

**Comment by smichel17** [comment]
Related to #260

**Comment by ddisanti** [comment]
@mpcjanssen I've installed the nodb version you provided and I think that while the spacing in the left drawer is fine, the spacing in the right drawer is very small and you have to tap carefully to choose the filter you want to activate. Even for a 5.7" screen device like mine.

**Comment by mpcjanssen** [comment]
@student agreed I will revert the right drawer change. I have the same issue

**Comment by mpcjanssen** [comment]
@ddisanti reverted right drawer spacing in 8a213276d4108855ddccae4626a0b9daeadb5295 . Updated debug builds on Dropbox.

**Comment by ddisanti** [comment]
@mpcjanssen I do not want to bother you, but I don't think you should revert the right drawer spacing back to its original size, but reduce it. As @vj777  said, the original spacing is large. What you did with the left drawer looks great.

**Comment by smichel17** [comment]
@ddisanti The original spacing basically said, "Ensure that each entry is always at least [some amount] high." The spacing change just removed that minimum requirement, so it depends on the size of the list items instead of some minimum size.

We should figure out what a good minimum size would be, and use that.
